synchronizer/synchronizer.go

```diff
@@ -196,12 +176,19 @@ func (s *ClientSynchronizer) syncTrustedState() error {
 		},
 		GlobalExitRootNum: new(big.Int).SetUint64(lastBatch.BatchNumber),
 	}
-
-	err = s.storage.AddTrustedGlobalExitRoot(s.ctx, ger, nil)
+	// If lastest ger it is already stored, skip
+	isAlreadyStored, err := s.storage.CheckTrustedExitRootExists(s.ctx, ger, nil)
```

Comment:
Instead of `CheckTrustedExitRootExists` , how about adding of the unique condition with `block_id, global_exit_root_num,  global_exit_root`

Reply:
We can no use a block_id because there is no block stored in the block table and globalexitrootNumber will disappear soon